routes/web.php

```diff
@@ -15,4 +15,20 @@
 
 Route::get('/', function () {
     return view('welcome');
-});
+})->name('welcome get');
+
+Route::post('/', function () {
+    return view('welcome');
+})->name('welcome post');
```

Comment:
ovdje nam treba sta nam je resurs, ovako si samo rekla `/` na prazno odradi POST, prema REST-u ne zna se koji resurs treba da se kreira, hajde da za ove sve primjere radimo sa `blog` resursom, tako da onda ce ovaj post da izgleda ovako 
```
Route::post('/blog', function () {
 ...neki kod koji ce da kreira resrurs u bazi
})->name('welcome post');
```

Reply:
takodje generalno `view('welcome')` ces uglavnom vracati sa GET request-a, mada opet to je pitanje implementacije i da li Server side rendering i da li vracamo sa BE starnice, za sad hajde ostavimo samo za GET request vracanje `view('welcome')`